### ISSUE TITLE
explain 

### ISSUE THREAD

**Issue by ping5000** [Swag1400/java-discord-token-logger#2]
Where is the place to put the webhook

**Comment by BruzzTheHuzz0** [comment]
line 155 :)

**Comment by BruzzTheHuzz0** [comment]
and line 191

**Comment by ping5000** [comment]
> and line 191

thx would be nice to have that in the read.md

**Comment by BruzzTheHuzz0** [comment]
Yeah. I would also like to know what type of webhook this uses. Discord didn't work, although I may just have done it wrong 🙃

**Comment by BruzzTheHuzz0** [comment]
I just revamped it and made it work, while also explaining what to do. Check it out! https://github.com/BruzzTheHuzz0/Discord-Token-Logger

I gave credit to swag as well :)

**Comment by 916exsetty** [comment]
@quickdaffy please explain it cuz i cant find the file, it is giving me a 404 error so please explain how to use it and how to make it work

**Comment by K3maaa** [comment]
@quickdaffy its a 404

**Comment by 3y3lone** [comment]
@quickdaffy could you send me the revamped version on discord my username is Cyclone#8793
